.github/CHANGELOG.md

```diff
@@ -374,8 +412,9 @@
 
 This release contains contributions from (in alphabetical order):
 
-Thomas Bromley, Christina Lee, Olivia Di Matteo, Anthony Hayes, Theodor Isacsson, Josh Izaac,
-Nathan Killoran, Shumpei Kobayashi, Romain Moyard, Maria Schuld, Antal Száva.
+Juan Miguel Arrazola, Thomas Bromley, Christina Lee, Alain Delgado Gran, Olivia Di Matteo, Anthony
+Hayes, Theodor Isacsson, Josh Izaac, Soran Jahangiri, Nathan Killoran, Shumpei Kobayashi, Romain
+Moyard, Zeyue Niu, Maria Schuld, Antal Száva.
```

Comment:
In included qchem contributors here as well (they are also included in the qchem changelog)

Reply:
Let me know if I'm missing anyone!